canonical-web-and-design/ubuntu.com
# Update payment summary and remove cart functionality

## Done
- Update payment summary section
- Add monthly subscription support
- Remove the cart from the advantage purchase flow
- Adjust the displayed cost in summary with 2 digits

## QA
- Go to https://ubuntu-com-9310.demos.haus/advantage/subscribe?test_backend=true

**Check the payment summary looks like [the design](https://app.zeplin.io/project/601d6513b99f24a8412ecfa5/screen/602ba60be0395723916f27ec)** 
- Select a desktop contract
- Add some quantity
- Select no support
- Check the "Your chosen plan" section matches [the design](https://app.zeplin.io/project/601d6513b99f24a8412ecfa5/screen/602ba60be0395723916f27ec)

**Removing the cart**
- See that switching between Monthly and Annual works as expected
- Check the payment modal appears in place of the cart
- Check the information is correct in the payment modal **I am yet to update the bottom of the payment modal**

**Disabling billing period**
- Cancel and change the support to standard and advanced
- See the summary switched to yearly and the option to change billing period is removed.
- Check the payment modal is correct

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3850
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3851
